multiraft/multiraft.go

```diff
@@ -30,7 +30,10 @@ import (
 	"golang.org/x/net/context"
 )
 
-const noGroup = uint64(0)
+const (
+	noGroup           = uint64(0)
+	eventBacklogLimit = 100000
```

Comment:
> // TODO(bdarnell): this is currently set high to avoid crashes when recovering a node. Since we don't send snapshots or chunk Append messages in such situations yet, the recovering node will receive a large amount of messages at the same time which can fill up this channel quickly.